gcp_variant_transforms/testing/integration/run_tests_common.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
```

Comment:
I wish there was a way that GitHub review interface would show me the diffs between this file and the old run_tests.py. Can you elaborate if you have done any specific changes or that this is mostly shuffling common functions?

Reply:
Yes, I mainly moved the common functions to this script. 
One change that worth mentioning is that the class `TestCase` is working as a base class, while `run_vcf_to_bq_tests` and `run_preprocessor_tests` provide their individual implementation of the method `validate_result`.